src/models/file.ts

```diff
@@ -1,13 +1,13 @@
 import multer from 'multer';
 import path from 'path';
-import { FILE_KEYS, ASSETS_DIR } from '../utils/constants';
+import { FILE_KEYS, Audios_DIR, Images_DIR } from '../utils/constants';
 import type { FileKeys } from '../utils/types';
 
 const regex = /^(\w{32})(\w{2})(.*)$/;
 
 const storage = multer.diskStorage({
   destination: (req, file, cb) => {
-    cb(null, ASSETS_DIR);
+    file.mimetype == 'audio/mpeg' ? cb(null, Audios_DIR) : cb(null, Images_DIR)
```

Comment:
You might want to take a more declarative approach. And I'd suggest you use the [file-type](https://www.npmjs.com/package/file-type) library to get the file type from the content. This is more tamperproof. 

```
function getUploadDirectory(fileData) {
  const fileType = FileType.fromBuffer(fileData);

  if (fileType) {
    if (fileType.mime.startsWith('audio/')) {
      return Audios_DIR;
    } else if (fileType.mime.startsWith('image/')) {
      return Images_DIR;
    }
  } else {
    // If file type detection fails or the type is not supported, reject the upload.
  }
}
```